examples/server/fieldmask_helper.go

```diff
@@ -39,7 +40,7 @@ func getField(obj interface{}, path string) (val reflect.Value) {
 		if v.Kind() == reflect.Ptr {
 			v = reflect.Indirect(v)
 		}
-		v = v.FieldByName(s)
+		v = v.FieldByName(generator.CamelCase(s))
```

Comment:
This is where I think the case fixing should go, in the user's server code. I had a quick go at doing something like golang/protobuf#457 (comment) to pull the name from the descriptor, but couldn't get it to work with a generic interface/reflection, I think it requires the concrete type to be passed in making this helper a lot less useful.